### DISCUSSION TITLE
Enable Interchain Accounts (ICA) So That Liquid Staking Protocols Can Onboard $AKT

### DISCUSSION THREAD

**Discussion by sneedpilled** [akash-network/community#480] [General]
Liquid staking protocols have revolutionized the Cosmos over the past year, allowing for countless new use cases. Akash should enable ICA so that liquid staking protocols such as Quicksilver and Stride can onboard $AKT into their respective protocols as $qAKT & $stAKT respectively. This would allow $AKT holders to dive deeper into Cosmos DeFi without forfeiting $AKT's now increased staking APR. Liquid staking is clearly an immensely popular service throughout the wider Cosmos, Akash should embrace this new liquid era.

**Comment by mosdcaing** [comment]
Absolutely! I couldn't agree more, especially considering the potential impact on $AKT and its ecosystem. Liquid staking protocols have the potential to significantly enhance the utility and liquidity of $AKT tokens, fostering a more dynamic and vibrant ecosystem. Embracing this new liquid era could truly propel $AKT to new heights and create exciting opportunities for its holders.

**Comment by hewi333** [comment]
Hi there @sneedpilled this is on radar of the core team, although in the backlog.

 https://github.com/akash-network/support/issues/171

**Comment by brewsterdrinkwater** [comment]
As @heyhewi mentioned, this is on the backlog in the roadmap for the core engineering team. You can stay up to date on the Akash Product and Engineering Roadmap [here](https://github.com/orgs/akash-network/projects/5/views/1).

**Comment by elix1er** [comment]
bump
